index.bs

```diff
@@ -838,6 +838,64 @@ interface ReceiveStream {
 ReceiveStream includes IncomingStream;
 </pre>
 
+# Protocol Mappings # {#protocol-mapping}
+
+This section specifies the QUIC protocol actions expected from methods defined
+in this specification, utilizing [[WEB-TRANSPORT-HTTP3]].
+
+  <table class="data">
+    <colgroup class="header"></colgroup>
+    <colgroup span=2></colgroup>
+    <thead>
+      <tr>
+        <th>Method</th>
+        <th>QUIC Protocol Action</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>{{IncomingStream/abortReading}}</td>
+        <td>send STOP_SENDING with code</td>
+      </tr>
+      <tr>
+        <td>{{OutgoingStream/abortWriting}}</td>
```

Comment:
Also, this top-level section links to `IncomingStream` and `OutgoingStream`, but doesn't mention datagrams, so it seems a bit unclear how broadly it applies.

I'm not sure how this structure would accommodate a solution to that.

Reply:
Bernard to open an issue on what happens if you abort or cancel datagrams